### PR TITLE
Kmohr-soprasteria/issue90

### PR DESCRIPTION
**Pull request by kmohr-soprasteria**

### CODE REVIEW

File: AdministratorAdministration.yaml [hunk 1]
```diff
@@ -69,20 +69,20 @@ paths:
...
+                            update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-c-nr-2-0-0-000/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-address/ip-address/ipv-4-address]'
                     domain-name:
                       type: string
                       pattern: '^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$'
                       description: >
                         'Domain name of application that shall be target of the handover process
-                        update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-c-im-nr-2-0-0-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/remote-address/domain-name]'
+                        update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-c-nr-2-0-0-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/remote-address/domain-name]'
```
Comment (IswaryaaS) [inline AdministratorAdministration.yaml:78]: line 78:
domain-name description is pointing to a http-client instance. This needs to be updated as 
"update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-c-nr-2-0-0-000/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-address/domain-name]"

**Review by IswaryaaS — CHANGES_REQUESTED** [review]
Changes regarding removing api-segment from uuid of http-c/s and tcp-c/s looks fine.
I provided a comment for a small update which is not related to this PR.
@openBackhaul , kindly requesting you to check.

File: AdministratorAdministration.yaml [hunk 2]
```diff
@@ -938,12 +938,12 @@ paths:
...
-                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address]'
+                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address]'
                         oam-approval-port:
                           type: integer
                           description: >
                             'Own TCP port
-                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/local-port]'
+                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/local-port]'
                       example:
```
Comment (IswaryaaS) [inline AdministratorAdministration.yaml:947]: As per comment https://github.com/openBackhaul/ExecutionAndTraceLog/pull/153#discussion_r1047382078, can we also update TAR data in line 1148 (example of /v1/list-applications)

File: AdministratorAdministration.yaml (hunk 1 above)
Comment (openBackhaul) [reply AdministratorAdministration.yaml:78]: Changed as proposed.
Thanks for indicating.

File: AdministratorAdministration.yaml (hunk 2 above)
Comment (openBackhaul) [reply AdministratorAdministration.yaml:947]: Changed protocol to HTTP, and adapted IP address and port to match with release number 1.0.0.

File: AdministratorAdministration.yaml [hunk 3]
```diff
@@ -69,20 +69,20 @@ paths:
...
+                            update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-c-nr-2-0-0-000/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-address/ip-address/ipv-4-address]'
                     domain-name:
                       type: string
                       pattern: '^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$'
                       description: >
                         'Domain name of application that shall be target of the handover process
-                        update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-c-im-nr-2-0-0-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/remote-address/domain-name]'
+                        update or create [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-c-nr-2-0-0-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/remote-address/domain-name]'
```
Comment (IswaryaaS) [inline AdministratorAdministration.yaml:78]: Here, the complete path needs to be updated to "tcp" as it is currently present as  http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration. This needs to be updated as tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**